partridge/gtfs.py

```diff
@@ -46,11 +51,15 @@ def get(self, filename: str) -> pd.DataFrame:
             df = self._cache.get(filename)
             if df is None:
                 df = self._read(filename)
-                df = self._filter(filename, df)
-                df = self._prune(filename, df)
-                self._convert_types(filename, df)
-                df = df.reset_index(drop=True)
-                df = self._transform(filename, df)
+                if self._proxy_feed:
```

Comment:
I'm not sure the choice should be to filter+prune OR convert+transform. Can you tell me a bit about how you are thinking about this behavior? I will need to think through the logic.

Reply:
As I see it for each table you filter you create a feed, and each feed is the source of the next one. except for the last layer the feeds (the proxy feeds) are only responsible for filtering the table according to the filter and the already filtered table (pruning). That's why those proxy feed you only need to filter and prune, before my change you did that by removing the transform and convert data from the configuration.
The last feed layer doesn't need to deal with the pruning and filtering since it doesn't even get a view as a parameter, and the lower level feeds are doing the pruning already.

Tell me if I missed something